static/bidder-info/zmaticoo_ssp.yaml

```diff
@@ -0,0 +1,10 @@
+aliasOf: "zmaticoo"
```

Comment:
@lxj15398019970  is zmaticoo an alias of yeahmobi?

Reply:
> @lxj15398019970 isn't zmaticoo an alias of yeahmobi?

No，This is a new company entity and brand。